### PR TITLE
Install Git [Link to Download] - Getting_Started.md

### LINKED ISSUE #584: Provide download link for Git
We should probably provide a link to the actual download site for Git [in `GETTING_STARTED.md`].

_Originally posted by @karnikaavelumani in https://github.com/EthanThatOneKid/acmcsuf.com/pull/577#discussion_r977856574_

### PR DESCRIPTION
**Pull request by erwinmedina**
Added a new item [line 4] to the getting started list within the Getting_Started.md file. I added Item2: Installing Git [Link to Git/Downloads]. I also adjusted the numbers listed after that so it would read [1,2,3,4,5,6,7] instead of [1,2,2,3,4,5,6].
Link Issue: [Issue #584](https://github.com/EthanThatOneKid/acmcsuf.com/issues/584)

### CODE REVIEW

**Review by k4rni — CHANGES_REQUESTED** [review]
Thank you so much Erwin for taking a look at this! There are a couple things that you can edit to make this PR even better in fact. An interesting thing about markdown is its ability to create ordered lists without having a numerical ascending, descending or random order. 
image

What this means is that instead of labeling all of the steps as:
```md
1. 
2. 
3. 
```

We would rather change it so that every single number in the list will begin with just 1 since numbers are automatically rendered no matter the order used in the markdown:
```md
1. 
1. 
1. 
```

Leaving them all as '1' would make it so that future updates to this list are as simple as appending a new line to the list instead of having to edit all the numbers individually.

If you would like more elaboration, feel free to take a look at this: https://www.markdownguide.org/basic-syntax/#ordered-lists
[Basic Syntax | Markdown Guide](https://www.markdownguide.org/basic-syntax/)

File: GETTING_STARTED.md [hunk 1]
```diff
@@ -1,12 +1,13 @@
 ## Getting started
 
 1. Install [Node 16 LTS](https://nodejs.org/en/)
-2. Install and get familiar with [Git](https://docs.github.com/en/get-started/quickstart/hello-world)
+2. Install [Git](https://git-scm.com/downloads)
+3. Install and get familiar with [Git](https://docs.github.com/en/get-started/quickstart/hello-world)
```
Comment (k4rni) [inline GETTING_STARTED.md]: I would also personally remove the "Install and" since it is already mentioned in the previous step to install Git

Comment (erwinmedina) [reply GETTING_STARTED.md]: That's a good call, i will make sure to remove that line, it was a little redundant. So sorry.

Comment (k4rni) [reply GETTING_STARTED.md]: Thank you so much!

**Review by k4rni — APPROVED** [review]
Thank you for all the help Erwin! Looks good to me:)

**Pull request merged**